<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FragmentationSolutionWindow</class>
 <widget class="QMainWindow" name="FragmentationSolutionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fragmentation Solution Algorithms</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button_Confirm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Back">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Algorithms">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Algorithms</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RadioButton_Compaction">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Compaction</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="RadioButton_Coalescing">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Coalescing</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
